src/main/java/codeu/model/data/Event.java

```diff
@@ -0,0 +1,173 @@
+
+package codeu.model.data;
+
+import java.time.Instant;
+import java.util.UUID;
+
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+import java.util.Locale;
+import java.time.ZoneId; 
+
+import codeu.model.store.basic.ConversationStore;
+import codeu.model.store.basic.MessageStore;
+import codeu.model.store.basic.UserStore;
+
+/** 
+ *  This class abstracts any object so it can be a User, Conversation, or Message. 
+ *  Made so a list can  easily hold and sort all types of objects at once.
+ */
+public class Event{
+
+	/* Denotes the type of event this is. 'c' for conversation, 'm' for message, and 'u' for user. */
+	private final Character eventType;
```

Comment:
Normally we use a ENUM for such thing like eventType.

Reply:
Will change this.